site/app/controllers/admin/ReportController.php

```diff
@@ -54,6 +54,7 @@ public function showReportPage() {
      * Generates grade summary files for every user
      *
      * @Route("/{_semester}/{_course}/reports/summaries")
+     * @Route("/api/{_semester}/{_course}/reports/summaries", methods={"POST"})
      */
     public function generateGradeSummaries() {
         if (!$this->core->getUser()->accessAdmin()) {
```

Comment:
`showError` function will die with a `Response::instance->render()` in the future.